packages/mobile/locales/en-US/global.json

```diff
@@ -125,6 +125,7 @@
   "odisInsufficientBalance": "Your balance is too low to complete this process",
   "matchmakingQuotaExceededError": "Matchmaking can be done only once",
   "paymentLimitReached": "The daily limit is {{dailyLimitcUSD}} cUSD ({{currencySymbol}}{{dailyLimit}})\n{{dailyRemainingcUSD}} cUSD ({{currencySymbol}}{{dailyRemaining}}) remaining",
+  "requestLimitError": "Can't request more than the {{limit}} cUSD daily limit.",
```

Comment:
Can we repurpose `paymentLimitReached` message to fit this?

Reply:
Do you mean the specific wording? `paymentLimitReached` ends with `{X} cUSD remaining` but in the case of requesting the remaining part doesn't make much sense I think, right?